x/authz/client/cli/tx.go

```diff
@@ -284,3 +300,15 @@ func bech32toValidatorAddresses(validators []string) ([]sdk.ValAddress, error) {
 	}
 	return vals, nil
 }
+
+func bech32toAccountAddresses(accAddrs []string) ([]sdk.AccAddress, error) {
```

Comment:
shall we move these to account types? 
```suggestion
func bech32toAccAddresses(accAddrs []string) ([]sdk.AccAddress, error) {
```